eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -0,0 +1,250 @@
+parameters:
```

Comment:
It would be nice to see a diff of this file compared to the one in azure-sdk-build-tools. I'm curious of what changes you needed to make.

Reply:
One way to accomplish that is for your first commit to be an unchanged version of this file and then make your edits to it as a second commit. That way we could look at the commits to see the changes as opposed to looking at the entire file as being new.

Reply:
I guess we don't want to add the signing step in the commit still. I will add the old file without that in an initial commit

Reply:
Added the original file with an initial commit.